src/component/action-card/action-card.component.jsx

```diff
@@ -2,18 +2,18 @@ import React from 'react'
 import * as ACS from './action-card.style'
 import Image from '../../asset/images/bicycle.jpg'
 
-export const ActionCard = () => {
+export const ActionCard = ({id}) => {
```

Comment:
please use the formatter, `CTRL + SHIFT + F`, it's not that difficult.
also make a note somewhere so I don't have to remind you so many times.